package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "ex-tutorial-nodejs",
-  "version": "0.9.1",
+  "version": "0.11.0",
```

Comment:
The version doesn’t really matter but we used to sync this up with daml-ledger until we forgot to bump it when we switched to `0.10.0`.